filebeat/module/elasticsearch/slowlog/ingest/pipeline-json.json

```diff
@@ -0,0 +1,108 @@
+{
+    "description": "Pipeline for parsing the Elasticsearch slow logs in JSON format.",
+    "on_failure": [
+        {
+            "set": {
+                "field": "error.message",
+                "value": "{{ _ingest.on_failure_message }}"
+            }
+        }
+    ],
+    "processors": [
+        {
+            "json": {
+                "field": "message",
+                "target_field": "elasticsearch.slowlog"
+            }
+        },
+        {
+            "drop": {
+                "if": "ctx.elasticsearch.slowlog.type != 'index_indexing_slowlog' && ctx.elasticsearch.slowlog.type != 'index_search_slowlog'"
+            }
+        },
+        {
+            "remove": {
+                "field": "elasticsearch.slowlog.type"
+            }
+        },
+        {
+            "rename": {
+                "field": "elasticsearch.slowlog.level",
+                "target_field": "log.level"
+            }
+        },
+        {
+            "rename": {
+                "field": "elasticsearch.slowlog.component",
+                "target_field": "elasticsearch.component"
+            }
+        },
+        {
+            "dot_expander": {
```

Comment:
Kind of odd that we need 2 steps here, but well.

Reply:
Agreed. This will help: https://github.com/elastic/elasticsearch/issues/36950.